README.md

```diff
@@ -31,6 +31,13 @@ these steps:
 If everything is configured right, the new release of your package should
 appear on GitHub and your package's website within a few minutes.
 
+> [!CAUTION]
+> We strongly recommend a 'dry run' the first time you use this workflow.
```

Comment:
I am not so sure about the "strongly": I am usually not doing that, the risks aren't that big. But I would indeed recommend it to anyone who does it the first time, or who does not feel sure enough of their own abilities and understanding of how all these things work and wants some safety.

```suggestion
> We recommend a 'dry run' the first time you use this workflow.
```